web-api/Cargo.toml

```diff
@@ -41,9 +45,7 @@ xayn-ai-bert = { path = "../bert" }
 xayn-ai-coi = { path = "../coi" }
 
 [dev-dependencies]
-bincode = { workspace = true }
-chrono = { workspace = true, features = ["clock", "serde"] }
-instant-distance = "0.6.0"
-ouroboros = "0.15.5"
-tokio = { workspace = true, features = ["sync"] }
 trycmd = "0.14.4"
+
+[features]
+mind = ["dep:bincode", "dep:chrono", "dep:instant-distance", "dep:ouroboros", "tokio/sync"]
```

Comment:
this is probably not the nicest solution, but the main issue comes from the fact that we can't simply make the routes generic over any `state: Data<AppState>` via traits because some of traits already implemented on the `AppState`s are not object safe.